survey-sdk/src/main/java/co/nimblehq/survey/sdk/network/NetworkBuilder.kt

```diff
@@ -31,6 +33,12 @@ open class NetworkBuilder {
         return this
     }
 
+
+    fun setToken(token: String): NetworkBuilder {
+        this.token = token
+        return this
+    }
```

Comment:
why do we need to set the token to the sdk. Should the sdk provide token to the app instead? 🤔

Reply:
@luongvo Yes we need. 
The SDK does not support authentication so far. Maybe we will do this on next phase I guess.